modules/etcd-defrag.adoc

```diff
@@ -29,6 +29,11 @@ Verify that the defragmentation process is successful by viewing one of these lo
 * cluster-etcd-operator pod
 * operator status error log
 
+[WARNING]
+====
+Automatic defragmentation can cause leader election failure in various OpenShift core components, such as the kube-controller-manager, which triggers a restart of that component. The restart is harmless and either triggers failover to the next running instance or the component resumes work again after the restart.
```

Comment:
```suggestion
Automatic defragmentation can cause leader election failure in various OpenShift core components, such as the `kube-controller-manager`, which triggers a restart of that component. The restart is harmless and either triggers failover to the next running instance or the component resumes work again after the restart.
```
Re: [Repo guidelines](https://github.com/openshift/openshift-docs/blob/37febdac457140d0a927f272910d5e879962d0ba/contributing_to_docs/doc_guidelines.adoc#api-object-formatting). 

Alternatively, if you want to avoid the backticks, you could use a general reference to the component, such as "Kubernetes controller manager." The Kubernetes docs have an example of both general and specific object usage: (https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/)

Reply:
Good catch/suggestion!